### ISSUE TITLE
Fibula fracture

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5888]
**Preferred term label:** Fibula fracture

**Synonyms** Fracture of the Fibula; Broken Fibula

**Definition (free text, please give PubMed ID)** Fracture of the fibula.

**Parent term (use hpo.jax.org/app)** Abnormality of fibula morphology HP:0002991; Bone fracture HP:0020110; Fractures of the long bones HP:0003084

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
